src/server/serverinventorymgr.h

```diff
@@ -36,19 +36,19 @@ class ServerInventoryManager : public InventoryManager
 		m_env = env;
 	}
 
-	Inventory *getInventory(const InventoryLocation &loc);
```

Comment:
Are these changes necessary? What's wrong with their implementations in `ServerInventoryManager`?

Reply:
We need make sure that the correct `getInventory` implementation is called, hence `virtual`. Furthermore, the scripting interface uses the server's `ServerInventoryManager`-type variable to retrieve inventories.

I will undo the other few function changes, though. They are currently not needed by the unittest anyway.